charts/reform-scan-blob-router/values.yaml

```diff
@@ -46,6 +46,8 @@ java:
 
     STORAGE_URL: https://reformscan.{{ .Values.global.environment }}.platform.hmcts.net
     STORAGE_BULKSCAN_URL: https://bulkscan.{{ .Values.global.environment }}.platform.hmcts.net
+    STORAGE_PCQ_URL: https://pcqshared{{ .Values.global.environment }}.blob.core.windows.net
```

Comment:
are we sure it'll have all these names?

Reply:
at the moment, storage account exists in aat only but it will be created with the same format

Reply:
need to be sure config is not used. well, not like used but it won't make any calls or smth. otherwise it'll fail

Reply:
pcq container is disabled so it'll not be used

Reply:
It'll be called only when dispatching the pcq envelopes